BunqSdk/Exception/ApiException.cs

```diff
@@ -1,32 +1,22 @@
-﻿using System.Collections.Generic;
-
+﻿
 namespace Bunq.Sdk.Exception
 {
-    /// <summary>
-    /// Exception triggered by API requests failed on the server side.
-    /// </summary>
     public class ApiException : System.Exception
     {
-        /// <summary>
-        /// Glue to concatenate the error messages.
-        /// </summary>
-        private const string GLUE_ERROR_MESSAGES = "\n";
-
+        private readonly string message;
         public int ResponseCode { get; private set; }
-        public IList<string> Messages { get; private set; }
 
-        /// <param name="responseCode">The HTTP Response code of the failed request.</param>
-        /// <param name="messages">The list of messages related to this exception.</param>
-        public ApiException(int responseCode, IList<string> messages) : base(
-            ConcatenateMessages(messages))
+        public override string Message
```

Comment:
Are you sure we need at all to define this property and the corresponding variable? Can't we just use the `Message` property of the base class here?

Reply:
consistency, all the others have it 🤔